forc/src/cli/commands/test.rs

```diff
@@ -90,5 +90,14 @@ pub(crate) fn exec(command: Command) -> Result<()> {
     out.lines().for_each(|line| println!("{}", line.unwrap()));
     thread.join().unwrap();
 
-    Ok(())
+    let child_success = match child.try_wait() {
+        Ok(Some(returned_status)) => returned_status.success(),
+        Ok(None) => child.wait().unwrap().success(),
+        Err(_) => false,
+    };
+
+    match child_success {
+        true => Ok(()),
+        false => Err(anyhow! {"child test process failed"}),
```

Comment:
@JoshuaBatty 
- This is the first time I'm seeing `anyhow! {` (space between and a bracket). I don't see it anywhere else in `sway/`. I feel like `clippy` should catch this?

Reply:
I wasn't sure about this one either actually, I have seen it in some other repo, but don't know whether it is a good way of doing this. `clippy` does not complain but I can convert it into a better/preferred version

Reply:
Let's wait for @mitchmindtree or @JoshuaBatty to weigh in.

Reply:
I also haven't seen this. I would prefer this to be changed to `anyhow!("child test process failed")` as it follows the syntax of how `anyhow!` is used throughout `sway`

Reply:
You could also simplify this to just `bail!("child test process failed")`